src/mn_setup.c

```diff
@@ -1733,7 +1735,7 @@ static setup_menu_t weap_settings3[] = {
     {"9Th Choice Weapon", S_WEAP | S_BOOM, M_X, M_SPC, {"weapon_choice_9"}},
     MI_GAP,
     {"Same Key Toggles Weapons", S_ONOFF | S_BOOM, M_X, M_SPC, {"doom_weapon_toggles"}},
-    {"Cycle Through All Weapons", S_ONOFF | S_BOOM, M_X, M_SPC, {"full_weapon_cycle"}},
```

Comment:
I believe this is still a setting that only works correctly with Boom or greater (look at P_PlayerThink), so the `| BOOM` needs to be there.

Reply:
I revert it: https://github.com/fabiangreffrath/woof/pull/2020/commits/7315f6bc8e0dbc25a404eb8f5ea5848f4b8693e9